src/events/ready.ts

```diff
@@ -72,11 +77,46 @@ export default {
         console.error(error);
       }
 
+<<<<<<< Updated upstream
+=======
```

Comment:
See above

Reply:
@Juilzze resolved this issue in latest commit.